workspace/__ardulib__/ESP32Servo/doxy.doxyfile

```diff
@@ -0,0 +1,2428 @@
+# Doxyfile 1.8.11
```

Comment:
Do we really need to include an 2,5k line .doxyfile?

Reply:
👌 
I'll remove it.